pkgs/by-name/da/daggerfall-unity/package.nix

```diff
@@ -0,0 +1,127 @@
+{
+  alsa-lib,
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchurl,
+  fetchzip,
+  lib,
+  libGL,
+  libXScrnSaver,
+  libXcursor,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXxf86vm,
+  libpulseaudio,
+  libudev0-shim,
+  makeDesktopItem,
+  nix-update-script,
+  stdenv,
+  vulkan-loader,
+  pname ? "daggerfall-unity",
+  includeUnfree ? false,
+}:
+let
+  docFiles =
+    [
```

Comment:
Q: What are the pros/cons of defining `docFiles` in a let-in block, vs having an overridable derivation attr and/or passthru attr?

Reply:
Not sure. If you think it'll be easier to understand, feel free to suggest a refactor. Otherwise I'd prefer to get this first version in.